getcov

```diff
@@ -112,7 +113,13 @@ exclude_data() {
 
   LCOV --remove "${LCOV_INFO}" "Developer/SDKs/*" -d "${OBJ_DIR}" -o "${LCOV_INFO}"
   LCOV --remove "${LCOV_INFO}" "main.m" -d "${OBJ_DIR}" -o "${LCOV_INFO}"
-  # Remove other patterns here...
+
+  #Remove anything the .xcodecoverageignore file has specified should be ignored.
+  (cat "${SRCROOT}/.xcodecoverageignore"; echo) | while read IGNORE_THIS; do
+    #use eval to expand any of the variables and then pass them to the shell - this allows
+  	#use of wildcards in the variables. 
```

Comment:
you have two spaces and then a tab to indent this line and the next.  I think you don't want the tabs.